addons/account/models/account_account.py

```diff
@@ -333,6 +333,7 @@ def _set_opening_debit_credit(self, amount, field):
                         field: amount,
                         'move_id': opening_move.id,
                         'account_id': self.id,
+                        'currency_id': self.currency_id.id or self.company_id.currency_id.id,
```

Comment:
`'currency_id': self.company_id.currency_id.id,` since you are not specifying any `amount_currency`